File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/comment/exception/RetrieveRandomCommentNotFoundException.java
package band.gosrock.domain.domains.comment.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class RetrieveRandomCommentNotFoundException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION =
            new RetrieveRandomCommentNotFoundException();

    private RetrieveRandomCommentNotFoundException() {
        super(CommentErrorCode.RETRIEVE_RANDOM_COMMENT_NOT_FOUND);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/comment/exception/RetrieveRandomCommentNotFoundException.kt
package band.gosrock.domain.domains.comment.exception

import band.gosrock.common.exception.DuDoongCodeException

class RetrieveRandomCommentNotFoundException : DuDoongCodeException(CommentErrorCode.RETRIEVE_RANDOM_COMMENT_NOT_FOUND) {
    companion object {
        val EXCEPTION: DuDoongCodeException = RetrieveRandomCommentNotFoundException()
    }
}
